components/movementsensor/imuwit/imu.go

```diff
@@ -177,66 +193,79 @@ func NewWit(
 	}
 
 	options := slib.OpenOptions{
-		BaudRate:        9600,
+		PortName:        conf.Port,
+		BaudRate:        115200,
 		DataBits:        8,
 		StopBits:        1,
 		MinimumReadSize: 1,
 	}
 
-	options.PortName = conf.Port
 	if conf.BaudRate > 0 {
 		options.BaudRate = uint(conf.BaudRate)
```

Comment:
While I'm looking at this line: if we need to convert it to a `uint` here, why not make it a `uint` in the `AttrConfig` to begin with? (It's possible that the answer is "because uints don't play nicely with JSON, and making that change will be a mess.")

Reply:
I'll explore the uint possibility on Monday. Maybe fine? Will merge without if it's not.

Reply:
uint works!